setupbase.py

```diff
@@ -262,7 +264,7 @@ def run(self):
             name, entrypt = script.split('=')
             name = name.strip()
             entrypt = entrypt.strip()
-            outfile = os.path.join(self.build_dir, name)
+            outfile = Path(os.path.join(self.build_dir, name))
```

Comment:
One of the goal of using Pathlib is to not have to use things like `os.path.join`, so a better way would be:
```suggestion
            outfile = Path(self.build_dir) / name
```

And even better woudl be to ensure that self.build_dir is already a path object, but that might be hard here.